packages/core/src/App/Components/Layout/Header/signup-button.jsx

```diff
@@ -19,14 +19,9 @@ const SignupButton = ({ className }) => {
     }, [isGBLoaded, trigger_os_signup]);
 
     const handleOutSystemsRedirection = () => {
-        switch (process.env.NODE_ENV) {
-            case 'production':
-                return 'https://hub.deriv.com/tradershub/signup';
-            case 'staging':
-                return 'https://staging-hub.deriv.com/tradershub/signup';
-            default:
-                return 'https://dev-hub.deriv.com/tradershub/signup';
-        }
+        if (getPlatformFromUrl().is_staging_deriv_app) return 'https://staging-hub.deriv.com/tradershub/signup';
```

Comment:
what happen with NODE_ENV check?, is it not reliable?

Reply:
No @amam-deriv it's not working properly so I decided to replace it with the safer one 😁